tests/store/test_footprints.py

```diff
@@ -727,7 +727,7 @@ def test_standardise_footprints_different_datasources(mock_metakeys):
     file2 = get_footprint_datapath("TAC-100magl_UKV_TEST_201608.nc")
 
     site = "TAC"
-    domain = "EUROPE"
+    domain = "EUROPE-test"
```

Comment:
Elsewhere the domain `test_europe` is used. Does this need to be the same?

Reply:
See Rachel's comment below